shared/src/jsMain/kotlin/com/macaosoftware/sdui/app/plugin/AuthPluginJsMain.kt

```diff
@@ -0,0 +1,71 @@
+package com.macaosoftware.sdui.app.plugin
+
+import com.macaosoftware.plugin.AuthPlugin
+import com.macaosoftware.plugin.LoginRequest
+import com.macaosoftware.plugin.LoginRequestForEmailWithLink
+import com.macaosoftware.plugin.LoginRequestForLink
+import com.macaosoftware.plugin.MacaoUser
+import com.macaosoftware.plugin.ProviderData
+import com.macaosoftware.plugin.SignupRequest
+import com.macaosoftware.plugin.util.MacaoResult
+
+class AuthPluginJsMain: AuthPlugin {
```

Comment:
This class should be in `auth-firebase-macao/jsMain`

Reply:
Ok